frontend/app/views/support/Dates.scala

```diff
@@ -59,18 +59,31 @@ object Dates {
 
   case class Range(start: String, end: String)
 
-  def dateRange(start: DateTime, end: DateTime): Range = {
-    def dateToks(dt: DateTime) = dt.toString("EEEE d MMMMM YYYY").split(" ").reverse
+  private def buildDateRange(start: DateTime, end: DateTime, fmt: String): Range = {
```

Comment:
@rtyley I pulled this out just because the code would've been repeated across `dateRange` and the new `shortDateRangeString`. Not quite sure what the code is doing to give it a meaningful name. Any thoughts on how to better refactor this appreciated.